### PR TITLE
Sana Ullah: Separate Response functions.

### PR DESCRIPTION
**Pull request by Sana-Ullah786**
This PR adds separate response functions in the Project. So, all responses should be consistent and have same format.
There are three response functions have been engineered in this PR.

- **custom_response(status_code,data, details)** The status code for this function and other two args can be tweaked.

### CODE REVIEW

File: src/responses.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from fastapi import status
+
+
+def success_response(data: object = None, details: str = "Succes") -> dict:
```
Comment (mustansirMuz) [inline src/responses.py:4]: A little typo, it should be "Success"

**Review by mustansirMuz — COMMENTED** [review]
I see the function error_response takes the status as an input parameter, which means that it can be used to return any type of response (even success), so why is there a need of another response specifically for success? We can have just one function, obviously renamed to something generic.

Also the function should return Response type object instead of a simple dictionary, so that it can be directly returned by the endpoint functions

File: src/responses.py (hunk 1 above)
Comment (Sana-Ullah786) [reply src/responses.py:4]: Fixed

**Review by mustansirMuz — APPROVED** [review]
LGTM

**Pull request merged**